src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -198,10 +198,7 @@ describe('ReactDOMComponent', function() {
 
       expect(console.error.argsForCall.length).toBe(1);
       expect(console.error.argsForCall[0][0]).toEqual(
-        'Warning: `span` was passed a style object that has previously been ' +
-        'mutated. Mutating `style` is deprecated. Consider cloning it ' +
-        'beforehand. Check the `render` using <span>. Previous style: ' +
-        '{fontSize: NaN}. Mutated style: {fontSize: NaN}.'
+        'Warning: `NaN` is an invalid value for the `fontSize` css style property'
```

Comment:
We actually should leave this message in tact and just update which call's arguments we look at - there will actually be 2 warnings fired, the first is your new warning and the 2nd is the one that was here. In this test we should look for the "modified" style object (even if it is a lie right now).